package.json

```diff
@@ -3,7 +3,8 @@
   "private": true,
   "module": "index.ts",
   "scripts": {
-    "dev": "turbo run dev --parallel"
+    "dev": "turbo run dev --parallel",
+    "turbo": "turbo"
```

Comment:
```suggestion
    "turbo": "turbo",
    "db:generate": "turbo db:generate",
```

Reply:
Good suggestion. I still wanted to access the `pakages/db`'s scripts without explicitly re-defining it here, but I think this will do. I used `bun turbo db:generate` instead before this PR, but this will be a bit cleaner.